h/streamer/views.py

```diff
@@ -10,6 +10,7 @@ def websocket_view(request):
     # Provide environment which the WebSocket handler can use...
     request.environ.update(
         {
+            "h.ws.debug": bool(request.params.get("debug")),
```

Comment:
Doesn't matter in practice becase no client is going to send any value for this but to avoid `"0"` being `True` you can use:


```diff
diff --git a/h/streamer/views.py b/h/streamer/views.py
index 3924d54b3..6683eea86 100644
--- a/h/streamer/views.py
+++ b/h/streamer/views.py
@@ -1,5 +1,6 @@
 from pyramid.view import forbidden_view_config, notfound_view_config, view_config
 from ws4py.exc import HandshakeError
+from pyramid.settings import asbool
 from ws4py.server.wsgiutils import WebSocketWSGIApplication
 
 from h.streamer import streamer, websocket
@@ -10,7 +11,7 @@ def websocket_view(request):
     # Provide environment which the WebSocket handler can use...
     request.environ.update(
         {
-            "h.ws.debug": bool(request.params.get("debug")),
+            "h.ws.debug": asbool(request.params.get("debug")),
             "h.ws.streamer_work_queue": streamer.WORK_QUEUE,
             "h.ws.identity": request.identity,
         }
```

Reply:
Will do, thanks.